integration/simple_app_test.go

```diff
@@ -73,5 +73,30 @@ func testSimpleApp(t *testing.T, context spec.G, it spec.S) {
 			Eventually(cLogs).Should(ContainSubstring("leftpad"))
 			Eventually(cLogs).Should(ContainSubstring("NPM_CONFIG_LOGLEVEL=error"))
 		})
+
+		it("builds an image with the SBOM", func() {
+			var err error
+			source, err = occam.Source(filepath.Join("testdata", "simple_app"))
+			Expect(err).NotTo(HaveOccurred())
+
+			image, _, err = pack.Build.
+				WithBuildpacks(nodeURI, buildpackURI, buildPlanURI).
+				WithPullPolicy("never").
+				Execute(name, source)
+			Expect(err).NotTo(HaveOccurred())
+
+			container, err = docker.Container.Run.
```

Comment:
In addition to explicitly checking the contents of this sbom file and implicitly checking for its presence on the filesystem, it would be good to explicitly verify the presence of the sbom files for the other two formats as similarly as possible to what's done [here](https://github.com/paketo-buildpacks/yarn-install/blob/065055ca40aa3316737e5b6d26a11c26f1e974f7/integration/simple_app_test.go#L85).

Reply:
Added additional tests